python/high-scores/high_scores.py

```diff
@@ -0,0 +1,21 @@
+def latest(scores):
+    return scores[-1]
+
+
+def personal_best(scores):
+    return max(scores)
+
+
+def personal_top_three(scores):
+    if len(scores) < 3:
+        scores.sort()
+        scores.reverse()
+        return scores
```

Comment:
As you may have seen in the internet, if you wish to return the sorted list, you can use `sorted` function like:  
```
return sorted(scores, reverse=True)
```